application.py

```diff
@@ -1,12 +1,116 @@
 #!/usr/bin/env python
 
+import logging
 import os
+from logging.handlers import RotatingFileHandler
 
-from app import create_app
-
+import watchtower
 from flask_script import Manager
 from flask_script import Server
+from structlog import configure
+from structlog.dev import ConsoleRenderer
+from structlog.processors import JSONRenderer, format_exc_info
+from structlog.processors import TimeStamper
+from structlog.stdlib import LoggerFactory, add_log_level
+from structlog.threadlocal import wrap_dict
+
+EQ_SR_LOG_GROUP = os.getenv('EQ_SR_LOG_GROUP', os.getenv('USER', 'UNKNOWN') + '-local')
+EQ_LOG_LEVEL = os.getenv('EQ_LOG_LEVEL', 'INFO')
+EQ_CLOUDWATCH_LOGGING = os.getenv("EQ_CLOUDWATCH_LOGGING", 'True').upper() == 'TRUE'
+EQ_WERKZEUG_LOG_LEVEL = os.getenv('EQ_WERKZEUG_LOG_LEVEL', 'INFO')
+EQ_DEVELOPER_LOGGING = os.getenv('EQ_DEVELOPER_LOGGING', 'False').upper() == 'TRUE'
+
+
+class StructlogStacktraceFormatter(logging.Formatter):
+    def format(self, record):
+        """
+        Log stack trace if exception key doesn't appear in log message. Logic taken from logging.Formatter.format.
+        """
+        record.message = record.getMessage()
+        if self.usesTime():
+            record.asctime = self.formatTime(record, self.datefmt)
+        s = self.formatMessage(record)
+        exception_key = '\"exception\":'
+        if record.exc_info and exception_key not in record.message:
+            # Cache the traceback text to avoid converting it multiple times
+            # (it's constant anyway)
+            if not record.exc_text:
+                record.exc_text = self.formatException(record.exc_info)
+        if record.exc_text:
+            if s[-1:] != "\n":
+                s += "\n"
+            s = s + record.exc_text
+        if record.stack_info:
+            if s[-1:] != "\n":
+                s += "\n"
+            s = s + self.formatStack(record.stack_info)
+        return s
```

Comment:
Removed this logic as it isn't required if we want to log the exception. We should add it to `logger.error(exc_info=e)`. We shouldn't use `log.exception` but we should produce the stack info and add it to `logger.error` so it will play nice with json messages